capabilityreceiver/pkg/capabilityhandler/capabilityreceiver.go

```diff
@@ -234,6 +237,64 @@ func saveMsgInDB(message []byte, args *map[string]interface{}) error {
 	return nil
 }
 
+func saveVersionResponseMsgInDB(message []byte, args *map[string]interface{}) error {
+	var err error
+	var existingEndpts []*endpointmanager.FHIREndpoint
+	var msgJSON map[string]interface{}
+
+	err = json.Unmarshal(message, &msgJSON)
```

Comment:
Instead of all of these arguments getting parsed out separately, we should just make an args struct like we do at the top of capabilityquerier/cmd/main.go